### PR TITLE
Add remove functionality

### PR DESCRIPTION
**Pull request by MargaretTechworld**
The following were added to this repository:

1. Implemented a function for adding a new task (when you click on the **enter key or the arrow icon**).
2. Implemented a function for deleting a task (when you click to edit the task the delete icon appears).
3. implemented a function to edit the task
4. implemented a function to update the index when the task is deleted
5. Added a descriptive Readme file
6. Added linters configuration
7. Added webpack configuration

Thank you for taking your time to review my code

### CODE REVIEW

File: src/modules/add.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+const displayList = document.querySelector('.todo-list');
+
+let toDoList = [];
+
+class TodoTask {
+  constructor(task, index = toDoList.length, description = false) {
+    this.task = task;
+    this.index = index;
```
Comment (DeliceLydia) [inline src/modules/add.js:9]: - [ ] The property index should be set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5). This should be applied as well when you delete

File: src/modules/add.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    toDoList.splice(index, 1);
+    this.updateIndexes();
+    this.saveToLocalStorage();
+  };
+
+  updateIndexes = () => {
+    for (let i = 0; i < toDoList.length; i += 1) {
+      toDoList[i].index = i;
+    }
+  };
```
Comment (DeliceLydia) [inline src/modules/add.js]: - [ ] A function for editing task descriptions should be implemented please work on that

**Review by DeliceLydia — APPROVED** [review]
## Hi @MargaretTechworld, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- I am able to add and delete a `todo`
- All changes to the To-Do List are being saved in local storage.
- Followed git workflow


## Required Changes ♻️
_Check the comments under the review._



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @MargaretTechworld, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Good UI design :white_check_mark: 
- [x] GitHub flow is used correctly :white_check_mark:  
- [x]  Todo tasks are editable :white_check_mark: 
- [x] Pull request has a descriptive title and summary :white_check_mark:  
- [x]  Task indexes are updated correctly as requested by the previous reviewer :white_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**